### ISSUE TITLE
Build is failing

### ISSUE THREAD

**Issue by sshehab1** [Azure/static-web-apps#1154]
Deployment Failed :(
Deployment Failure Reason: Failed to deploy the Azure Functions.

**Comment by MahrRah** [comment]
Same here

**Comment by benscattergood-empyrean** [comment]
I'm experiencing the same, no code change. My API functions are also no longer working.

**Comment by DaanStreng** [comment]
Same problem, whole of production down. Spend an hour getting azure functions out of our static web app to get it working again.

**Comment by ElliotGibson** [comment]
Also getting this with our SWA deployments for nextjs:

```bash
Running 'yarn run build'...

yarn run v1.22.15
$ echo 'restoring the framework'
restoring the framework
Done in 0.06s.
Preparing output...

Copying files to destination directory '/23e3617a-d9cd-43aa-8bce-f8db9d94dbf9-swa-oryx/api'...
Done in 2 sec(s).

Removing existing manifest file
Creating a manifest file...
Manifest file created.
Copying .ostype to manifest output directory.

Done in 19 sec(s).


---End of Oryx build logs---
Function Runtime Information. OS: linux, Functions Runtime: ~4, node version: 16
Finished building function app with Oryx
Zipping Api Artifacts
Done Zipping Api Artifacts
Zipping App Artifacts
Done Zipping App Artifacts
Uploading build artifacts.
Finished Upload. Polling on deployment.
Status: InProgress. Time: 0.1271505(s)
Status: InProgress. Time: 15.2650373(s)
Status: InProgress. Time: 30.3016765(s)
Status: Failed. Time: 45.4484438(s)
Deployment Failed :(
Deployment Failure Reason: Failed to deploy the Azure Functions.

For further information, please visit the Azure Static Web Apps documentation at https://docs.microsoft.com/en-us/azure/static-web-apps/
If you believe this behavior is unexpected, please raise a GitHub issue at https://github.com/azure/static-web-apps/issues/
Exiting
##[error]Error: The process '/usr/bin/bash' failed with exit code 1
Finishing: AzureStaticWebApp
```

This if attempt #3 of trying to redeploy our app

**Comment by joeklin** [comment]
We are static only. No functions and receiving the following

```
No Api directory specified. Azure Functions will not be created.
Either no Api directory was specified, or the specified directory was not found. Azure Functions will not be created.
We are currently experiencing problems communicating with our content server. Please try again later or file an issue if this behavior continues.
```

**Comment by mishapos** [comment]
We have investigated the issue and found there was an outage with the Azure Functions powering your app's dynamic backends. This caused deployments and the "APIs" blade in the Azure Portal to fail. The underlying Azure Functions issue has been mitigated. Deployments and the APIs blade in the Azure Portal should both be operational again.

**Comment by DaanStreng** [comment]
@mishapos we're still experiencing the issue. Deploy error from azure devops release street "Deploy Azure Static Web Apps"  is:
`2023-05-04T06:46:42.6034455Z ---End of Oryx build logs---
2023-05-04T06:46:42.6064327Z ##[warning]Function Runtime Information. OS: linux, Functions Runtime: ~4, node version: 16
2023-05-04T06:46:42.6069492Z Finished building function app with Oryx
2023-05-04T06:46:42.7840478Z Zipping Api Artifacts
2023-05-04T06:46:43.2775521Z Done Zipping Api Artifacts
2023-05-04T06:46:43.5609092Z Zipping App Artifacts
2023-05-04T06:46:46.5796029Z Done Zipping App Artifacts
2023-05-04T06:46:46.5796701Z Uploading build artifacts.
2023-05-04T06:46:47.5325641Z ##[section]Finished Upload. Polling on deployment.
2023-05-04T06:46:47.6828981Z Status: InProgress. Time: 0.1500905(s)
2023-05-04T06:47:02.8307689Z Status: Failed. Time: 15.2968345(s)
2023-05-04T06:47:02.8307931Z ##[error]Deployment Failed :(
2023-05-04T06:47:02.8308158Z ##[error]Deployment Failure Reason: Failed to deploy the Azure Functions.
2023-05-04T06:47:02.8462731Z 
2023-05-04T06:47:02.8463554Z ##[warning]For further information, please visit the Azure Static Web Apps documentation at https://docs.microsoft.com/en-us/azure/static-web-apps/
2023-05-04T06:47:02.8464047Z ##[warning]If you believe this behavior is unexpected, please raise a GitHub issue at https://github.com/azure/static-web-apps/issues/
2023-05-04T06:47:02.8464242Z ##[warning]Exiting
2023-05-04T06:47:03.0988315Z ##[error]Error: The process '/usr/bin/bash' failed with exit code 1
2023-05-04T06:47:03.1026999Z ##[section]Finishing: Static Web App: `

**Comment by mbalslow** [comment]
+1

**Comment by mishapos** [comment]
Yes, there is another functions outage. Will post updates here once it has been mitigated.

**Comment by joeklin** [comment]
Is there a reason these function outages are not being reported [here](https://azure.status.microsoft/en-us/status)  ?

**Comment by richardseftonTNP** [comment]
Again. Literally none of our deployed Static Web Apps are working atm.

**Comment by honzakostejn** [comment]
I'm experiencing the same issue.

I'm able to track the latest successful run of the managed azure function (static web app) to 2023/4/22. After that I can't provide better estimation to when this has stopped working.

<img width="1679" alt="image" src="https://user-images.githubusercontent.com/32060653/236612666-a1de6725-8173-4ed9-bff9-fa5cf2214a5f.png">

Should I switch to "bring your own" or there's mitigation for this planned?

**Comment by y2kbugger** [comment]
Our local `swa deploy` succeeds without error, but I am getting all of these over the last day:

- 401 (empty body)
- 401 (iis style error as html body)
- 403 (azure style error as html body)
- 503 (plaintext "Function host is not running")

the 4XX happened to an existing SWA, and the 503 happened after recreating the SWA in the portal as we were troubleshooting.

**Comment by y2kbugger** [comment]
not sure i this is a clue, but saw this in the logs, this is a dotnet-isolated app
![image](https://github.com/Azure/static-web-apps/assets/6101677/1c34b584-f6f7-46b1-916e-53738e55fc8d)

**Comment by y2kbugger** [comment]
For 503 mitigation is adding these two setting (with your proper values of course) to your swa-cli config:

    "apiLanguage": "dotnetisolated",
    "apiVersion": "7.0",

For the 401, the temporary mitigation is to set the function AuthorizationLevel to Anonymous:

    public async Task<HttpResponseData> GetValue([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequestData req)
